server/controllers/backup.js

```diff
@@ -42,7 +42,10 @@ module.exports = exports = async (req, res, next) => {
 		});
 	}
 	catch(error) {
-		log.error({error});
+		log.error({
+			route: req.route.path,
```

Comment:
These aren't easily searchable in the codebase though

Reply:
OK will do again

Reply:
Ideally, some sort of searchable string would be good, even if it’s something like the function name uppercased

Reply:
REQUEST_COMPLETED_MATE